joern-cli/frontends/kotlin2cpg/src/main/scala/io/joern/kotlin2cpg/ast/AstForDeclarationsCreator.scala

```diff
@@ -276,7 +276,7 @@ trait AstForDeclarationsCreator(implicit withSchemaValidation: ValidationMode) {
         astForIfAsExpression(expr.getInitializer.asInstanceOf[KtIfExpression], None, None)
       } else {
         val assignmentNode   = operatorCallNode(Operators.assignment, s"$tmpName = ${rhsCall.getText}", None)
-        val assignmentRhsAst = astsForExpression(rhsCall, None).head
+        val assignmentRhsAst = astsForExpression(rhsCall, None).headOption.getOrElse(Ast())
```

Comment:
Perhaps following the above convention on line 78
```suggestion
        val assignmentRhsAst = astsForExpression(rhsCall, None).headOption.getOrElse(Ast(unknownNode(rhsCall, "<empty>"))
```